ruby/vampires.rb

```diff
@@ -0,0 +1,43 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+until employees == 0
+	puts "What is your name?"
+	name = gets.chomp.downcase
+	puts "How old are you? What year were you born?"
+	age = gets.chomp
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?(y/n)"
+	garlic_bread = gets.chomp
+	puts "Would you like to enroll in the company's health insurance?(y/n)"
+	health_insurance = gets.chomp
+
+	if (age.to_i < 120) && (garlic_bread == 'y' || health_insurance == 'y')
+		puts "Probably not a vampire."
+	#if i put "Probably a vampire" first it will always go to that result instead of
+	#"Almost certainly a vampire."
+	elsif age.to_i >= 120 && garlic_bread == 'n' && health_insurance == 'n'
+		puts "Almost certainly a vampire."
+	elsif (age.to_i >= 120) && (garlic_bread == 'n' || health_insurance == 'n')
+		puts "Probably a vampire."
+	elsif name == "drake cula" || name == "tu fang"

```

Comment:
I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.

```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```